### PR TITLE
Add some basic admin API docs

### PR DESCRIPTION
**Pull request by erikjohnston**

### CODE REVIEW

File: docs/admin_api/purge_remote_media.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Purge Remote Media API
+======================
+
+The purge remote media API allows server admins to purge old cached remote
+media. 
+
+The API is::
+
+    POST /_matrix/client/r0/admin/purge_history/
```
Comment (ara4n) [inline docs/admin_api/purge_remote_media.rst]: really? surely purge_media or something?

**Comment by ara4n** [comment]
why are these endpoints on the CS API if they're synapse-specific admin?

File: docs/admin_api/purge_remote_media.rst (hunk 1 above)
Comment (erikjohnston) [inline docs/admin_api/purge_remote_media.rst]: Ooops, good spot. Fixed

**Comment by erikjohnston** [comment]
The admin API has always been there. I can move them I suppose, or doc them as `/_matrix/client/unstable/admin/...` ?

**Comment by ara4n** [comment]
let's move it in future. this LGTM otherwise.

**Pull request merged**